docker-compose.yaml

```
@@ -264,6 +264,7 @@ services:
       interval: 15s
       timeout: 10s
       retries: 8
+    #entrypoint: ['tail', '-f', '/dev/null']
```

YAMLlint (1.37.1)

[warning] comment not indented like content

(comments-indentation)